src/modules/shared/components/ShareFolderBlade/ShareFolderBlade.tsx

```diff
@@ -30,13 +30,15 @@ const ShareFolderBlade: FC<ShareFolderBladeProps> = ({ isOpen, onClose, folderId
 		handleSubmit,
 		formState: { errors },
 		control,
+		resetField,
 	} = useForm<ShareFolderBladeFormState>({
 		resolver: yupResolver(SHARE_FOLDER_FORM_SCHEMA()),
 	});
 
 	const handleClose = () => {
 		setEmailInputValue('');
 		setValue('email', '');
+		resetField('email');
```

Comment:
do we still need the setValue here, if we reset the field anyways?
```suggestion
		resetField('email');
```